sklearn_numba_dpex/kmeans/kernels/compute_euclidean_distances.py

```diff
@@ -15,16 +17,18 @@ def make_compute_euclidean_distances_fixed_window_kernel(
     n_samples,
     n_features,
     n_clusters,
-    preferred_work_group_size_multiple,
-    centroids_window_width_multiplier,
-    centroids_window_height,
+    sub_group_size,
     work_group_size,
     dtype,
 ):
+    centroids_window_height = work_group_size // sub_group_size
```

Comment:
Could you please add an inline comment to explain this?

Reply:
It's added in `lloyd_single_step.py`